src/main/java/com/aws/greengrass/certificatemanager/certificate/CertificateExpiryMonitor.java

```diff
@@ -62,7 +62,7 @@ void watchForCertExpiryOnce() {
                 continue;
             }
             try {
-                cg.generateCertificate(connectivityInfoProvider::getCachedHostAddresses);
+                cg.generateCertificate(connectivityInfoProvider::getCachedHostAddresses, "certificate has expired");
```

Comment:
realized that this isn't necessarily true since we can rotate before expiry. to prevent confusion, will distinguish between expired and almost expired